static/app/components/platformPicker.tsx

```diff
@@ -107,13 +107,18 @@ class PlatformPicker extends Component<PlatformPickerProps, State> {
     const filtered = tempSelectablePlatforms
       .filter(this.state.filter ? subsetMatch : categoryMatch)
       .sort((a, b) => {
-        if (startsWithPunctuation(a.name) && !startsWithPunctuation(b.name)) {
+        // Treat Nuxt as 'Vue' to sort it alongside Nuxt
+        // Other related frameworks are sorted next to each other automatically (solid - solidstart, svelte - sveltekit, ...)
```

Comment:
I recently updated the platform list alphabetically. You can check the changes in [this PR](https://github.com/getsentry/sentry/pull/74832). With these updates, Nuxt won't be next to Next. Why did we decide to do that?

Reply:
See the discussion above :)
https://github.com/getsentry/sentry/pull/78467#discussion_r1784441878

Reply:
Yes I saw that afterward, but I am still not convinced 😬 let discuss that internally